tgui/packages/tgui/interfaces/SeedFabricator.js

```diff
@@ -0,0 +1,172 @@
+import { useBackend, useLocalState } from "../backend";
+import { Blink, Button, Collapsible, Flex, Icon, Modal, NumberInput, ProgressBar, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { pluralize } from "./common/stringUtils";
+
+const DefaultSort = {
+  Fruit: 1,
+  Vegetable: 2,
+  Herb: 3,
+  Flower: 4,
+  Miscellaneous: 5,
+};
+
+const SeedsPerRow = 3;
+
+export const SeedFabricator = (props, context) => {
+  const { data } = useBackend(context);
+  const { canVend, isWorking, maxSeed, name, seedCount } = data;
+  const categories = data.seedCategories || [];
+
+  categories.sort((a, b) => (DefaultSort[a.name] || 6) - (DefaultSort[b.name] || 6));
```

Comment:
why 6 instead of defaultsort.Other or something

Reply:
Changed it to sort categories based on either their `DefaultSort` value, or alphabetically.